doc/nrf/libraries/nrf_security/doc/driver_config.rst

```diff
@@ -87,33 +87,46 @@ Key Derivation Function
 
 To enable key derivation function (KDF) support, set one or more of the following Kconfig options:
 
-+--------------------------+------------------------------------------------------------+
-| KDF algorithm            | Configuration option                                       |
-+==========================+============================================================+
-| HKDF                     | :kconfig:option:`CONFIG_PSA_WANT_ALG_HKDF`                 |
-+--------------------------+------------------------------------------------------------+
-| TLS 1.2 PRF              | :kconfig:option:`CONFIG_PSA_WANT_ALG_TLS12_PRF`            |
-+--------------------------+------------------------------------------------------------+
-| TLS 1.2 PSK to MS        | :kconfig:option:`CONFIG_PSA_WANT_ALG_TLS12_PSK_TO_MS`      |
-+--------------------------+------------------------------------------------------------+
-| TLS 1.2 EC J-PAKE to PMS | :kconfig:option:`CONFIG_PSA_WANT_ALG_TLS12_ECJPAKE_TO_PMS` |
-+--------------------------+------------------------------------------------------------+
++--------------------------+---------------------------------------------------------------+
+| KDF algorithm            | Configuration option                                          |
++==========================+===============================================================+
+| HKDF                     | :kconfig:option:`CONFIG_PSA_WANT_ALG_HKDF`                    |
++--------------------------+---------------------------------------------------------------+
+| PBKDF2_HMAC              | :kconfig:option:`CONFIG_PSA_WANT_ALG_PBKDF2_HMAC`             |
```

Comment:
I think it is more natural to put `PBKDF2-HMAC`. This is how the PSA documentation describes it in text.

Reply:
fixed